### PR TITLE
add all EMAIL environment variables in Values

### PR DESCRIPTION
**Pull request by Tutuchan**
Hi @pmint93, thanks for this repo, it's really helpful :).

I added all the email-related environment variables in the deployment, please let me know if I need to do anything more ? I'll probably wait until #20 is merged to update the chart version again.

Best,
Pierre

### CODE REVIEW

File: charts/metabase/templates/deployment.yaml [hunk 1]
```diff
@@ -144,6 +144,42 @@ spec:
...
+            value: {{ .Values.email.security | quote }}
+          {{- end }}
+          {{- if .Values.email.username }}
+          - name: MB_EMAIL_SMTP_USERNAME
+            value: {{ .Values.email.username | quote }}
+          {{- else if or .Values.email.existingSecret .Values.email.existingSecretUsernameKey -}}
+          - name: MB_EMAIL_SMTP_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ or .Values.email.existingSecret (printf "%s-email" (include "metabase.fullname" .)) }}
```
Comment (pmint93) [inline charts/metabase/templates/deployment.yaml:170]: it seems you are trying to make the default value for `.Values.email.existingSecret` ? If so, just [Using the default function](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) and update README.md accordingly

Comment (pmint93) [reply charts/metabase/templates/deployment.yaml]: And yeah, now I look at how `database.existingSecret` is done and didn't know why I let it be, maybe too lazy to deeply reviewed like this now :D

Comment (Tutuchan) [reply charts/metabase/templates/deployment.yaml]: > What you think ? If README is not an ideal location, may be values.yaml (but commented out) ?
Yes, I think it'd be better. I'll add a comment

> And yeah, now I look at how database.existingSecret is done and didn't know why I let it be,
I can update this part as well if you want !

File: charts/metabase/values.yaml
Comment (sralla-eq) [inline charts/metabase/values.yaml:1]: Should you remove the comments on the Horizontal scalability of Metabase based on https://github.com/metabase/metabase/issues/2754

**Pull request closed without merging**